src/card.css

```diff
@@ -0,0 +1,87 @@
+.card {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
```

Comment:
do not set width directly to bem blocks, use bem mixes instead

Reply:
fixed